README.md

```diff
@@ -77,3 +77,18 @@ Required: `false`<br>
 Default: `warn`
 
 When a lookup cannot be resolved, this specifies whether to throw an error or log a warning. In the case of a warning, the invalid lookup value will be replaced with an empty string.
+
+### lookupPattern
+
+Type: `RegExp` <br>
+Required: `false` <br>
+Default: `/@\(?([a-z-]+)\)?\b/g`
+
+The regular expression by which the property will be searched
+
+For example, to enable working with [postcss-custom-media](https://github.com/postcss/postcss-custom-media), look up only css-properies @font-size, @color, ...:
```

Comment:
I'm not sure I understand the relation to `custom-media` here?

Reply:
Is the goal to whitelist certain properties? Or blacklist them? Maybe it would be better to have an array:

```js
['color', 'font-size`]
```

Reply:
What if i use @ in in many places?

I mean

color: @(max-width: 200px) red; ------> postcss-custom-media

line-height: @font-size; ------> postcss-property-lookup

You have in example:
```
line-height: 1.5;
font-size: @(line-height)em;
```

And this is a good option, but i want use both plugins (property-lookup and custom-media).
And use them both, I have to write a specific property, without brakets
For those who do not need it will not change anything

Reply:
But whitelist/blacklist, it a good option too

Reply:
Is custom media not like this?

```css
@custom-media --small-viewport (max-width: 30em);
```

That would be easier to ignore.

Reply:
Oh, wrong plugin -_-.

https://github.com/dimitrinicolas/postcss-inline-media#postcss-custom-media

Reply:
padding: 20px @(--small-viewport) 10px;

Conflicts with your plugin.

And if I write a pattern on which I look for properties, there will be no conflicts.

The goal is precisely this

Reply:
```
a {
	color: red;
	background-color: @color @(max-width: 200px) yellow;
}
```